_posts/2015-05-22-about-the-quality-and-fun.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: About the quality and fun!
+author: ruoat
+excerpt: Software quality is important. But let's not forget the fun!
+---
+
+About the quality
+-----------------
+
+Two weeks ago there was annual Solita Codecamp where developer-oriented people can try out and play with not so enterprisey software than we do usually every day.
+
+This year’s task was to implement a [Shopping Bot](https://github.com/janneri/bots-of-black-friday/) and let it compete against other Codecampers’ bots and drink some refreshments. While there was an [example](https://github.com/janneri/bots-of-black-friday/tree/master/example-java-bot) of basic bot implemented with [Java](http://hmpg.net/) and [Spring Boot](http://projects.spring.io/spring-boot/) coders were allowed to pick the \[favourite|most hated|most interesting\] language to accomplish the task. You would see others’ progress and implemented bots’ shopping frenzy in real time from the screen. And that freedom brought an interesting matter about the software development. Would you implement the QuickN’DirtyBot to take quick wins while others are still coding or would you take more time and implement the most robust UltimateBotWithBellsAndWhistles2000, which would win all others after it is implemented. Or just try something with some exotic language and have fun while you learn something new. A few took the shortest way with Java, someone wanted code [Haskell](https://www.haskell.org/), one group had perversion for [Elixir](http://elixir-lang.org/), some wanted to fiddle with [Clojure](http://clojure.org/) and of course there was [Node.js](https://nodejs.org/) bot, too.
```

Comment:
juoko botti eka lauseessa? :)

an example of *a* basic bot

brought *up* an interesting matter,  varmaankin?

there was *a* Node.s bot, too